package.json

```diff
@@ -46,6 +46,7 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "2.1.1",
     "ember-cli-uglify": "^2.0.0",
+    "ember-cli-update": "^0.15.0",
```

Comment:
I'd prefer to use the global installation of this, since we don't need to install it for CI systems etc.

Reply:
Alright, commit removed.